diff --git a/setup.py b/setup.py
index 09c8650..5a7b566 100644
--- a/setup.py
+++ b/setup.py
@@ -13,27 +13,22 @@ setup(
     author='teedrz',
     packages=['lietorch'],
     ext_modules=[
-        CUDAExtension('lietorch_backends', 
+        CUDAExtension('lietorch_backends',
             include_dirs=[
-                osp.join(ROOT, 'lietorch/include'), 
+                osp.join(ROOT, 'lietorch/include'),
                 osp.join(ROOT, 'eigen')],
             sources=[
-                'lietorch/src/lietorch.cpp', 
+                'lietorch/src/lietorch.cpp',
                 'lietorch/src/lietorch_gpu.cu',
                 'lietorch/src/lietorch_cpu.cpp'],
             extra_compile_args={
-                'cxx': ['-O2'], 
+                'cxx': ['-O2'],
                 'nvcc': ['-O2',
-                    '-gencode=arch=compute_60,code=sm_60', 
-                    '-gencode=arch=compute_61,code=sm_61', 
-                    '-gencode=arch=compute_70,code=sm_70', 
-                    '-gencode=arch=compute_75,code=sm_75',
-                    '-gencode=arch=compute_75,code=compute_75',
-                    
+                    '-gencode=arch=compute_86,code=sm_86',
                 ]
             }),
 
-        CUDAExtension('lietorch_extras', 
+        CUDAExtension('lietorch_extras',
             sources=[
                 'lietorch/extras/altcorr_kernel.cu',
                 'lietorch/extras/corr_index_kernel.cu',
@@ -43,18 +38,11 @@ setup(
                 'lietorch/extras/extras.cpp',
             ],
             extra_compile_args={
-                'cxx': ['-O2'], 
+                'cxx': ['-O2'],
                 'nvcc': ['-O2',
-                    '-gencode=arch=compute_60,code=sm_60', 
-                    '-gencode=arch=compute_61,code=sm_61', 
-                    '-gencode=arch=compute_70,code=sm_70', 
-                    '-gencode=arch=compute_75,code=sm_75',
-                    '-gencode=arch=compute_75,code=compute_75',
-                    
+                    '-gencode=arch=compute_86,code=sm_86',
                 ]
             }),
     ],
     cmdclass={ 'build_ext': BuildExtension }
 )
-
-
